keras/src/layers/convolutional/conv2d.py

```diff
@@ -7,10 +7,10 @@ class Conv2D(BaseConv):
     """2D convolution layer.
 
     This layer creates a convolution kernel that is convolved with the layer
-    input over a 2D spatial (or temporal) dimension(height and width) to produce a tensor of
-    outputs. If `use_bias` is True, a bias vector is created and added to the
-    outputs. Finally, if `activation` is not `None`, it is applied to the
-    outputs as well.
+    input over a 2D spatial (or temporal) dimension(height and width) to 
+    produce a tensor ofoutputs. If `use_bias` is True, a bias vector is created 
+    and added to the outputs. Finally, if `activation` is not `None`, it is 
+    applied to the outputs as well.
 
```

Comment:
Not an expert but imho this would be more clear for most people:

> This layer creates a kernel that is convolved over the 2D spatial dimensions (height and width) of the input..